ISISComputingGroup/EPICS-inst_servers
# Dataweb: Blockserver now serves up current config on a URL

Shows relevant information on a web page.

To test:
* Start up instrument and navigate to localhost:8008 in a browser
* Confirm that the page updates when configs change (requires a refresh)
